simpleflow/swf/executor.py

```diff
@@ -1148,6 +1148,8 @@ def propagate_signals(self):
 
         known_workflows_ids = frozenset(known_workflows_ids)
 
+        signals_scheduled = False
+
         for signal in history.signals.values():
             input = signal['input']
```

Comment:
minor: `input` is a built-in function.

Reply:
Yep; but this is one of those names we use everywhere 🙄

Reply:
Ok no pb.